tasks/Debian.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Add apt-key
-  apt_key: id=E56151BF keyserver=keyserver.ubuntu.com state=present
+  apt_key: id=DF7D54CBE56151BF keyserver=keyserver.ubuntu.com state=present
```

Comment:
This is the full key ID for the Mesosphere repo.

http://keyserver.ubuntu.com/pks/lookup?op=get&search=0xDF7D54CBE56151BF